routes/wishlist.js

```diff
@@ -0,0 +1,96 @@
+const express = require('express');
+const router = express.Router();
+const Wishlist = require('../models/wishlist.model');
+const checkAuth = require('../middlewares/check-auth');
+
+const getWishlist = (req, res, next) => {
+    const userId = req.userId;
+
+    Wishlist.getWishlist(userId).then(results => {
+        res.status(200).json({
+            success: true,
+            wishlist: results
+        });
+    }).catch(err => {
+        res.status(500).json({
+            error: true, 
+            message: "Database error!"
+          })
+    });
+};
+
+const addWishlist = (req, res, next) => {
+    const userId = req.userId;
+    const postUserId = req.body.wishlist.userId;
+    if(userId != postUserId) {
+        return res.status(401).json({
+            error: true,
+            message: "Not authorised to perform this operation"
+        })
+    }
```

Comment:
add this check to get and delete too

Reply:
no its not needed

Reply:
yes , sorry about that , I should have looked correctly